src/calculateRentalCost.js

```diff
@@ -3,8 +3,20 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE = 40;
+  const sum = days * PRICE;
+
+  if (days >= 7) {
+    return sum - 50;
+  }
+
+  if (days >= 3) {
+    return sum - 20;
```

Comment:
Similarly to the previous comment, the number '3' and the discount value '20' are hardcoded. Replace them with constants that describe what they represent.